CMakeLists.txt

```diff
@@ -159,6 +167,7 @@ set(lib_SRCS
     lib/events/directchatevent.cpp
     lib/events/encryptionevent.cpp
     lib/events/encryptedevent.cpp
+    lib/events/roomkeyevent.cpp
```

Comment:
Should also come to `libquotient.pri`?

Reply:
Thank you! You right, added header and source path to `.pri`